default/python/AI/AIDependencies.py

```diff
@@ -20,6 +20,8 @@
     my_industry = AIDependencies.INDUSTRY_PER_POP * my_population
 """
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
+from common.configure_logging import convenience_function_references_for_logger
+(debug, info, warn, error, fatal) = convenience_function_references_for_logger(__name__)
```

Comment:
May be we can import this https://docs.python.org/2/library/logging.html#logging.Logger.info  and use it?

I wan to have option to import single log function:  `from common.configure_logging import debug`, but this will require to play with frames to get proper filename.